tools/monitoring/main.go

```diff
@@ -113,14 +113,20 @@ func hello(w http.ResponseWriter, r *http.Request) {
 
 func testCloudSQLConn(w http.ResponseWriter, r *http.Request) {
 	log.Printf("Serving request: %s", r.URL.Path)
-	log.Println("Testing mysql database connection.")
+	fmt.Fprintf(w,"Testing mysql database connection...")
 
-	err := dbConfig.TestConn()
+	db, err := dbConfig.Connect()
 	if err != nil {
-		fmt.Fprintf(w, "Failed to ping the database %v", err)
-		return
+		log.Fatalln(err)
```

Comment:
shouldnt we write the err message on the response?

Reply:
since we are doing the checking through the url in browser. It's easier to see it printed on browser instead of in console. If in the future, we make it into some automated test, then it's better to put it in console.

Reply:
In this case, I think you meant to use `fmt.Fprintf(w, ...` as that makes the response viewable through the browser. [log.Fatalln](https://golang.org/pkg/log/#Fatalln) prints to the log and exits the app. I think for checking the alert condition, exiting the app after error may not be ideal.

Reply:
@yt3liu You are right. I read the change reversely ;)